omnibus/files/private-chef-cookbooks/private-chef/recipes/config.rb

```diff
@@ -37,6 +37,13 @@
   node.consume_attributes({"previous_run" => old_config['private_chef']})
 end
 
+# used for checking if LDAP and SAML are both enabled
+chef_manage = { 'chef_manage' => {} }
+if ::File.exists?('/var/opt/chef-manage/etc/chef-manage-running.json')
+  chef_manage = JSON.parse(IO.read("/var/opt/chef-manage/etc/chef-manage-running.json"))
+  node.consume_attributes({"chef_manage" => chef_manage['chef-manage']})
```

Comment:
If you move this node.consume outside of the `if` then you can avoid checking that `node["chef-manage"]` exists before using it.